website/content/partials/known-issues/transit-managed-keys-panics.mdx

```diff
@@ -0,0 +1,16 @@
+### Transit Encryption with Cloud KMS managed keys causes a panic
+
+#### Affected versions
+
+- 1.13.1+ up to 1.13.7 inclusively
+- 1.14.0+ up to 1.14.3 inclusively.
```

Comment:
```suggestion
- 1.14.0+ up to 1.14.3 inclusively
```

Style correction: format all line items consistently

Reply:
Applied in https://github.com/hashicorp/vault/pull/23080/commits/73a206f089239be8eee21737a8db955c6bbe762b